delly_1.0.3/README.md

```diff
@@ -0,0 +1,81 @@
+# CWL and Dockerfile for running Delly
+
+## Version of tools in docker image (/container/Dockerfile)
+
+| Tool	| Version	| Location	|
+|---	|---	|---	|
+| ubuntu  	| 18.04  	|  -	|
```

Comment:
The image is not based on ubuntu right? can we correct this?

Reply:
I believe the containers are ubuntu based: https://github.com/msk-access/cwl-commandlinetools/blob/dellytools1.0.3/delly_1.0.3/container/Dockerfile#L4/ I believe I did this because the official delly image is ubuntu base. But I can switch to something lighter weight like centos?

Reply:
Ahh i got confused by `-platform=linux/amd64` do you know what difference does it make if specify `-platform=linux/amd64 ` and when we dont

Reply:
I think it might be from the template I used. I agree that it isn't the most useful. I'm fine cutting it if we want. 

I believe the `-platform=linux/amd6` solves an issue for building certain docker images on mac m1s: https://stackoverflow.com/questions/65612411/forcing-docker-to-use-linux-amd64-platform-by-default-on-macos

Reply:
Good to know lets keep it !